manifests/params.pp

```diff
@@ -184,6 +183,12 @@
   $server_http_port           = 8139
   $server_http_allow          = []
 
+  # use puppetserver (JVM) or puppet master (Ruby)?
+  $server_implementation = $aio_package ? {
+    true    => 'puppetserver',
+    default => 'master',
+  }
+
```

Comment:
The issue I have here is that nothing can override $aio_package, right? Is there a ::globals thing like postgres has?